lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7669,6 +7670,19 @@ ValueDecl *ClangImporter::Implementation::importBaseMemberDecl(
   return known->second;
 }
 
+bool ClangImporter::Implementation::isClonedBaseMemberDecl(ValueDecl *decl) {
+
+  // If this is a cloned decl, we don't want to reclone it
```

Comment:
I think a comment explaining why we only want to do this for decls **without** a clang node would be useful